branchsearch.cpp

```diff
@@ -234,8 +249,12 @@ int main(int argc, char** argv){
                 if(selected_branch > 0) selected_branch--;
             }else if (c == KEY_DOWN){
                 if(selected_branch < filtered_branches.size()-1 && selected_branch < MAX_BRANCH_SHOWN - 1) selected_branch++;
+            }else if (c == KEY_DC){
```

Comment:
Supporting backspace can't hurt

Reply:
Backspace is reserved for typing

Reply:
But couldn't it be used if nothing was typed so far? I thought about using it but your point is fair.

Reply:
Mmm is think that’s kind of arbitrary and confusing